addons/account/models/account.py

```diff
@@ -1134,6 +1143,8 @@ class AccountReconcileModel(models.Model):
         ('fixed', 'Fixed'),
         ('percentage', 'Percentage of amount')
         ], string="Second Amount type",required=True, default='percentage')
+    second_tax_price_include = fields.Boolean(string='Is Second Tax Price Include', related='second_tax_id.price_include')
+    second_tax_include = fields.Boolean(string='Second Tax Include')
```

Comment:
use the same label as price_include field in account.tax => 'Tax Included in Price'